readthedocs/rtd_tests/tests/test_privacy.py

```diff
@@ -15,24 +15,13 @@
 
 
 class PrivacyTests(TestCase):
-    def tearDown(self):
-        tasks.update_docs = self.old_bd
+    fixtures = ["eric"]
 
     def setUp(self):
-        self.eric = User(username='eric')
-        self.eric.set_password('test')
-        self.eric.save()
-
-        self.tester = User(username='tester')
-        self.tester.set_password('test')
-        self.tester.save()
```

Comment:
That is undoing the refactor from #1532
See https://github.com/rtfd/readthedocs.org/pull/1532/files#diff-27d2585b4d0b054a45164644132a4e5eL17